docs/database/sql-server.md

```diff
@@ -152,6 +154,37 @@ and SHOULD be provided **at span creation time** (if provided at all):
 <!-- END AUTOGENERATED TEXT -->
 <!-- endsemconv -->
 
+## Database Context Propagation
+
+**Status**: [Development][DocumentStatus]
+
+### SET CONTEXT_INFO
+
+Instrumentations MAY make use of [SET CONTEXT_INFO](https://learn.microsoft.com/en-us/sql/t-sql/statements/set-context-info-transact-sql?view=sql-server-ver16) to add text format of [`traceparent`](https://www.w3.org/TR/trace-context/#traceparent-header) before executing a query. This is an opt-in behavior that SHOULD be explicitly enabled by the user.
+
```

Comment:
could we provide some guidance for tracestate and/or baggage? Or should we say that it's not possible to propagate them?

Reply:
We could say it is not possible to propagate them due to the 128-byte length limit.

I want to avoid giving users the illusion that they can propagate variable-length variables, like tracestate and baggage.

Reply:
sure, let's mention that propagation is limited to traceparent (or similar) due to length limit concerns and instrumentations SHOULD NOT propagate tracestate or baggage